random-reviewer-picker/main.js

```diff
@@ -0,0 +1,142 @@
+const github = require("@actions/github");
+const core = require("@actions/core");
+const { random } = require("lodash");
+
+/**
+ * Fetch and transform inputs into usable variables
+ **/
+function getInputs() {
+  const reviewer_list = core
+    .getInput("reviewer_list")
+    .split(",")
+    .map(i => i.replace(" ", ""));
+  const reviewer_amount = parseInt(core.getInput("reviewer_amount"));
+  const maintainer_list = core
+    .getInput("maintainer_list")
+    .split(",")
+    .map(i => i.replace(" ", ""));
+  const maintainer_amount = parseInt(core.getInput("maintainer_amount"));
+  const skip_busy = !!core.getInput("skip_busy");
+
+  const GITHUB_TOKEN = core.getInput("GITHUB_TOKEN");
+
+  return {
+    reviewer_list,
+    reviewer_amount,
+    maintainer_list,
+    maintainer_amount,
+    skip_busy,
```

Comment:
Is there any reason to use snake_case instead of camelCase ?
For JS naming consistency, i think we can stick to camelCase. What do you think ?

Reply:
The github API uses snake case, but I don't have a strong opinion on that, I will switch to camelCase